src/main/java/org/sagebionetworks/bridge/models/upload/Upload.java

```diff
@@ -44,6 +45,13 @@ static Upload create() {
     /** @see #getHealthCode */
     void setHealthCode(String healthCode);
 
+    /**
+     * Metadata fields for this upload, as submitted by the app. This corresponds with the
+     * uploadMetadataFieldDefinitions configured in the app.
+     */
+    JsonNode getMetadata();
+    void setMetadata(JsonNode metadata);
+
```

Comment:
However, we're not validating these align at the time an upload request is received...?